static/templates/settings/account-settings.handlebars

```diff
@@ -199,13 +199,17 @@
         <div id="get_api_key_box">
             <p>{{t "Please re-enter your password to confirm your identity." }}
               <a href="/accounts/password/reset/" target="_blank">{{t "Never had one? Forgotten it?" }}</a></p>
-            <form cslass="form-horizontal" id="get_api_key_form">
-                <div class="control-group">
-                    <label for="password" class="control-label">{{t "Current password" }}</label>
-                    <input type="password" autocomplete="off"
-                      name="password" id="get_api_key_password" value="" />
-                    <button type="submit" name="view_api_key" id="get_api_key_button"
-                      class="button sea-green">{{t 'Get API key' }}</button>
+            <form class="form-horizontal" id="get_api_key_form">
+                <div class="control-group flex-container">
+                    <div class="lab">
```

Comment:
Can you find a better convention for `lab`? Also, is there a particular reason why you created two new `div` elements for the elements instead of applying `flex-grow` to the submit button?

Reply:
Sure I will find a better name for "lab".
I wanted 'label' and 'input' fields to be one entity and button to be another entity placed below them, and that's why created two divs to align them easily and add optimum space between (button) and (label + input) fields.
Initially, I wanted to have all the elements in a single line in the smaller screen, but the view would have looked congested.
Will be waiting for further comments and then will change the name of the class from "lab" to something better.

Reply:
Can you try using the `flex-grow` approach to have the button take up all its bottom space so that the input/label can be on the same line? Let me know if it doesn't work out, though.

Reply:
@synicalsyntax 
Are you telling me to try flex-grow approach in this redesigned form or in the original version?
Means removing the two extra divs (lab and btn) and using flex grow on the original form's button?